### ISSUE TITLE
Change in Theme

### ISSUE THREAD

**Issue by AnaHotaling** [realbig/RBM_TH_EPIC#42]
Question: what would be involved in changing the Word Press theme used by Epic to one that is more mobile optimized? 

To get an idea of what we're envisioning, can you please go to http://www.i5publishing.com/ on your smartphone? We would love something cool like that for the Epic mobile site.

**Comment by brashrebel** [comment]
To change themes would mean we'd be redeveloping almost all the custom work we've ever done on this site. Mostly stuff like the custom events menu, the events page layout, the staff layout, the home page, etc. All the custom stuff like that which we've done over the past couple years will not exist in a new theme and would have to be created again, differently to fit within the new design. It would be a big project.

I don't want to discourage from that idea though. It has been a couple years and I think a fresh new design every 2 or 3 years is a smart practice. If you were serious about this project we could begin exploring a bit some alternative layouts and I could estimate a project cost.

However, if your only motive is to improve the experience for users on smaller screens then I would recommend you simply open up issues for every specific problem you see and we'll see what we can do to address them. I'm certain that is the simpler of the two options.

**Comment by AnaHotaling** [comment]
I agree completely about refreshing the look of a site every two or three
years, since the freshness of a site does reflect the freshness of the
company that it represents. As for just fixing all the mobile issues, it's
not quite that easy. Eva approached me, asking what I thought about having
an Epic Races app developed. She wanted something snazzy and sharp looking
that people could download onto their phones, because most of our users are
apparently mobile users. I responded that this is a great idea, but it
would also mean that we would constantly be doing updates with each
operating system update, that each time we did something to the site it
would undoubtedly mean that the app would have to be updated, and that we
would pretty much have to keep a programmer/colder on retainer to
continually do these updates to the app for us. In other words, it would be
an added hassle. I recommended just making the mobile version of our site
jazzier and flashier, since we would be able to do the updates to this
ourselves. I showed her that site I shared with you – it's my publishers
site — and she really liked how flashy it was and told me to go ahead and
work with you guys to see about getting a jazzy mobile version of our site
that is optimized completely for mobile users.

No way the two can have separate themes but draw on the same content, huh?

I will let her know that it would involve a total redesign of the website,
but then also let her know that it's about time to do that, anyway,
especially if she wants to change the front pictures.

On Friday, February 5, 2016, Kyle notifications@github.com wrote:

> To change themes would mean we'd be redeveloping almost all the custom
> work we've ever done on this site. Mostly stuff like the custom events
> menu, the events page layout, the staff layout, the home page, etc. All the
> custom stuff like that which we've done over the past couple years will not
> exist in a new theme and would have to be created again, differently to fit
> within the new design. It would be a big project.
> 
> I don't want to discourage from that idea though. It has been a couple
> years and I think a fresh new design every 2 or 3 years is a smart
> practice. If you were serious about this project we could begin exploring a
> bit some alternative layouts and I could estimate a project cost.
> 
> However, if your only motive is to improve the experience for users on
> smaller screens then I would recommend you simply open up issues for every
> specific problem you see and we'll see what we can do to address them. I'm
> certain that is the simpler of the two options.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/realbig/RBM_TH_EPIC/issues/42#issuecomment-180359438.

## 

Ana Hotaling
Director of Inbound Marketing, Epic Races
p: 734-585-7101 a: 600 S. Wagner Rd., Ann Arbor, MI 48103
s: http://www.epicraces.com e: ana@epicraces.com
http://facebook.com/epicracesevents http://epicraces http://epicraces

https://newoldstamp.com/editor/

**Comment by brashrebel** [comment]
Well hold on there! It has actually been on my to-do list to call Eva and discuss the possibility of developing an Epic Races app. It is no longer strictly true that a mobile app _must_ operate separately from a website. It is now possible to develop an app in such a way that it's data is drawn from the same source as the website which means your ongoing content management remains the same as it is now. I'd like to actually open up that discussion if Eva is willing to have it.

As for a separate theme for mobile visitors, I would not advise that. This site is extremely customized with tons of unique elements and we'd have to go through every single thing on the site and determine what should be on the mobile version, how it should look, where it should go, etc. Also, any time you have a new idea for adding more information or a new feature, we'd have to do that work on both the mobile version and the large screen version thereby doubling the cost.

In summary, our options are:
A. Just continue working together to make the current theme's experience on small screens better.
B. Redesign the entire site on a new theme that is simpler and more mobile friendly to begin with.
C. (can be done with A or B) Develop an Epic Races mobile app.
D. Retire early and move to Costa Rica.

**Comment by AnaHotaling** [comment]
I vote for option D!

Although I am kind of partial to Tahiti…

If the app would draw its content directly from the site, and I am all for
the app! That was actually my first response to her: yes, an Epic app! And
then I started thinking. Thinking can be bad. My chief concern was having
to retain a programmer and continually deal with bugs that would come up as
operating systems update and update and update.

I can broach this with Eva, since last night she asked me to find out if
I'd heard anything from you about redoing the mobile site. I was supposed
to meet with her today at 1:30 in the office, but of course 1:45 PM if you
only opening my son's doctor has.

Thanks!

On Friday, February 5, 2016, Kyle notifications@github.com wrote:

> Well hold on there! It has actually been on my to-do list to call Eva and
> discuss the possibility of developing an Epic Races app. It is no longer
> strictly true that a mobile app _must_ operate separately from a website.
> It is now possible to develop an app in such a way that it's data is drawn
> from the same source as the website which means your ongoing content
> management remains the same as it is now. I'd like to actually open up that
> discussion if Eva is willing to have it.
> 
> As for a separate theme for mobile visitors, I would not advise that. This
> site is extremely customized with tons of unique elements and we'd have to
> go through every single thing on the site and determine what should be on
> the mobile version, how it should look, where it should go, etc. Also, any
> time you have a new idea for adding more information or a new feature, we'd
> have to do that work on both the mobile version and the large screen
> version thereby doubling the cost.
> 
> In summary, our options are:
> A. Just continue working together to make the current theme's experience
> on small screens better.
> B. Redesign the entire site on a new theme that is simpler and more mobile
> friendly to begin with.
> C. (can be done with A or B) Develop an Epic Races mobile app.
> D. Retire early and move to Costa Rica.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/realbig/RBM_TH_EPIC/issues/42#issuecomment-180403477.

## 

Ana Hotaling
Director of Inbound Marketing, Epic Races
p: 734-585-7101 a: 600 S. Wagner Rd., Ann Arbor, MI 48103
s: http://www.epicraces.com e: ana@epicraces.com
http://facebook.com/epicracesevents http://epicraces http://epicraces

https://newoldstamp.com/editor/

**Comment by AnaHotaling** [comment]
The answer is... theme change! Eva asked me to start looking for themes that are mobile optimized. She wants the focus to be on mobile. 

Do you need me to forward her email?

No word about the app. My guess is site first, then reflect the site design on the app?

Guess I'll be spending tonight at WordPress,org. 😊

**Comment by brashrebel** [comment]
@AnaHotaling some discussion may be necessary for this. It probably won't be as simple as finding one that exists which you like and then having us take the time to customize it to include all of your custom needs. Your needs are quite extensive and unique and it may be necessary to develop something totally from scratch. But then we'll cross that bridge when we get there. For now, just get some inspiration and I'll help explain what's possible.

**Comment by AnaHotaling** [comment]
Sounds good to me! Don't worry... I did tell Eva it wasn't just picking a
new theme and transferring the content instantly. 😊

On Saturday, February 6, 2016, Kyle notifications@github.com wrote:

> @AnaHotaling https://github.com/AnaHotaling some discussion may be
> necessary for this. It probably won't be as simple as finding one that
> exists which you like and then having us take the time to customize it to
> include all of your custom needs. Your needs are quite extensive and unique
> and it may be necessary to develop something totally from scratch. But then
> we'll cross that bridge when we get there. For now, just get some
> inspiration and I'll help explain what's possible.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/realbig/RBM_TH_EPIC/issues/42#issuecomment-180875637.

## 

Ana Hotaling
Director of Inbound Marketing, Epic Races
p: 734-585-7101 a: 600 S. Wagner Rd., Ann Arbor, MI 48103
s: http://www.epicraces.com e: ana@epicraces.com
http://facebook.com/epicracesevents http://epicraces http://epicraces

https://newoldstamp.com/editor/

**Comment by AnaHotaling** [comment]
So I have now gone through every single available theme and I have a list of a dozen or so that I feel may be options. They were all classified as mobile friendly or mobile optimized. Of course I only got to see what they look like on desktop. I even tried looking on my iPad and iPhone but these still only gave me the desktop view. Any suggestions on how to see these themes in their mobile view?

**Comment by brashrebel** [comment]
If you're viewing an actual demo of a theme, you can use Chrome's device emulation tool. When you're viewing a page, right click somewhere on the page and hit Inspect Element. Some new panels will open up. Look for a little tiny cell phone icon and click on that. This will present the site to you in a new grid as it would be viewed on a specific device which you can select in the upper left. Let me know if you have trouble.

**Comment by AnaHotaling** [comment]
Ummmmmm....

On a Mac, so no right clicking. And on Firefox... I haven't ever used 
Chrome! I'll give this a shot tonight when things will be supposedly 
calmer. But be prepared for me to be back going, ACK!

Or hopefully more like "These are the five themes I've narrowed down to, 
what do you think?" :)

---

Ana Hotaling
ana@hotalings.com

---

**Comment by d4mation** [comment]
In Firefox you'll be looking for this button here:

![image](https://cloud.githubusercontent.com/assets/7770631/13009411/38022528-d16a-11e5-8793-5bb4a4518625.png)

On a Mac, you can execute a Right-Click by pressing CTRL+Click I believe. You can also turn on Right Clicking for the Touchpad (If you're running a MacBook) within the System Settings.

You should be able to get into these menus without right-clicking by pressing F12 in both browsers.

**Comment by AnaHotaling** [comment]
Great! Will give it a shot once I actually wake my brain up. Thanks!!!

On Friday, February 12, 2016, Eric Defore notifications@github.com wrote:

> In Firefox you'll be looking for this button here:
> 
> [image: image]
> https://cloud.githubusercontent.com/assets/7770631/13009411/38022528-d16a-11e5-8793-5bb4a4518625.png
> 
> On a Mac, you can execute a Right-Click by pressing CTRL+Click I believe.
> You can also turn on Right Clicking for the Touchpad (If you're running a
> MacBook) within the System Settings.
> 
> You should be able to get into these menus without right-clicking by
> pressing F12 in both browsers.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/realbig/RBM_TH_EPIC/issues/42#issuecomment-183350993.

## 

Ana Hotaling
Director of Inbound Marketing, Epic Races
p: 734-585-7101 a: 600 S. Wagner Rd., Ann Arbor, MI 48103
s: http://www.epicraces.com e: ana@epicraces.com
http://facebook.com/epicracesevents http://epicraces http://epicraces

https://newoldstamp.com/editor/
